tracee-ebpf/tracee/events_processor.go

```diff
@@ -246,6 +250,32 @@ func (t *Tracee) removeSocketTail() {
 	}
 }
 
+func (t *Tracee) removePipeTail() {
+	// removing sys_pipe_exit_tail from sys_exit_tails.
+	bpfMap, err := t.bpfModule.GetMap("sys_exit_tails")
```

Comment:
I suggest here to check if err != nil and return then continue with your logic. By the way, maybe we can add error return to the function?

Reply:
because we are using sync.Once - we can't return a value from the function (as far as i understand it). the way to do that is to set a global variable to hold the value we want to return.
but i guess in this case we can just use the t.handleError().